index.html

```diff
@@ -185,42 +211,44 @@ <h1 class="project-name">Base Apparel coming soon page</h1>
           <div class="fnl-btn">
             <button type="button" class="btn" disabled>See project</button>
           </div>
+          </div>
         </div>
       </section>
       <section class="visit">
-        <h2 class="cnct">CONTACT US</h2>
+        <h2 class="cnct1">CONTACT US</h2>
+        <h2 class="cnct2">Let's Chat</h2>
         <p class="note">If you have an application
           you are interested in developing
           , a feature that you need built
           or a project that needs coding.
-          I’d love to help with it
+          I'd love to help with it
         </p>
         <form action="https://formspree.io/f/mgedkakg" method="POST" class="prjct">
-          <input class="wrt" maxlength="30" type="text" placeholder="Enter your name" required>
-          <input class="wrt" type="Email" placeholder="Enter your mail" required>
+          <input class="wrt" maxlength="30" type="text" placeholder=" Enter your name" required>
+          <input class="wrt" type="Email" placeholder=" Enter your mail" required>
           <textarea class="textarea" maxlength="500"  name="comment"
-            placeholder="Enter your text"></textarea>
-          <button class="btn" type="submit">Submit</button>
+            placeholder=" Enter your text"></textarea>
+          <button class="btn frm " type="submit">Send</button>
         </form>
       </section>
       <section class="final">
-        <ul class="icon">
-          <li>
+        <ul class="icon1 ">
+          <li class="icon-item">
             <a href="https://github.com/DanUshindi">
               <i class="fab fa-github"></i>
             </a>
           </li>
-          <li>
+          <li class="icon-item">
             <a href="https://www.linkedin.com/in/dan-ushindi-821415215/">
               <i class="fab fa-linkedin-in"></i>
             </a>
           </li>
-          <li>
+          <li class="icon-item">
             <a href="#">
               <i class="fab fa-angellist"></i>
             </a>
           </li>
-          <li>
+          <li class="icon-item">
             <a href="https://twitter.com/dan_ushindi">
               <i class="fab fa-twitter"></i>
             </a>
```

Comment:
- [ ] Please make sure that the mobile design also looks as in the template. Currently, the social icons are displayed in a column, while in the template they are arranged on a single row.

![](https://i.imgur.com/09GYV1k.png)